core/src/main/java/com/confighub/core/utils/FileUtils.java

```diff
@@ -200,33 +161,32 @@ private static String previewFile(final Context context,
                     if (null == injectFile)
                         value = "[ ERROR: No file resolved ]";
                     else
-                        value = previewFile(context, injectFile.getContent(), resolved, passwords, breadcrumbs);
+                        value = renderFileContents(context, injectFile.getContent(), passwords, resolvedProperties, breadcrumbs);
                 }
                 else
                 {
                     if (property.isEncrypted())
                     {
                         String spName = property.getPropertyKey().getSecurityProfile().getName();
-                        if (null != passwords && passwords.containsKey(spName))
-                        {
-                            String password = passwords.get(spName).get(0);
-                            if (!Utils.isBlank(password))
-                                property.decryptValue(password);
-                        }
+                        if (passwords.containsKey(spName))
```

Comment:
what's the goal here?  seems like we are adjusting this slightly for blank passwords

Reply:
So the passwords map will never be null, based on how this function is called. This function combines the functionality of the old resolveFile()/previewFile() functions. In resolveFile(), there's no extra check for if the password is blank.